web/core/components/issues/issue-layouts/list/base-list-root.tsx

```diff
@@ -112,20 +112,24 @@ export const BaseListRoot = observer((props: IBaseListRoot) => {
     [fetchNextIssues]
   );
 
-  //kanbanFilters and EIssueFilterType.KANBAN_FILTERS are used becuase the state is shared between kanban view and list view
-  const handleCollapsedGroups = (toggle: "group_by" | "sub_group_by", value: string) => {
-    if (workspaceSlug) {
-      let collapsedGroups = issuesFilter?.issueFilters?.kanbanFilters?.[toggle] || [];
-      if (collapsedGroups.includes(value)) {
-        collapsedGroups = collapsedGroups.filter((_value) => _value != value);
-      } else {
-        collapsedGroups.push(value);
+  // kanbanFilters and EIssueFilterType.KANBAN_FILTERS are used becuase the state is shared between kanban view and list view
+  const handleCollapsedGroups = useCallback(
+    (toggle: "group_by", value: string) => {
```

Comment:
you can also remove `toggle` argument completely since it is always "group_by" for list

Reply:
Done, made the changes.